packages/flutter_splash_tizen/example/pubspec.yaml

```diff
@@ -0,0 +1,28 @@
+name: flutter_splash_tizen_example
+description: Demonstrates how to use the flutter_splash_tizen plugin.
+
+publish_to: 'none'
+
+environment:
+  sdk: ">=2.12.0 <3.0.0"
+
+dependencies:
+  flutter:
+    sdk: flutter
+
+  flutter_splash_tizen:
+    path: ../
+
+  cupertino_icons: ^1.0.2
+
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
+
+  flutter_lints: ^1.0.0
+
+flutter:
+  uses-material-design: true
+
+flutter_splash_tizen:
+  image: assets/test.png
```

Comment:
no new line at the end of file

Reply:
one more comment - I assume that the example application is after adding a splash file with flutter-tizen pub run flutter_splash_tizen:create because related line is added to tizen-manifest.xml. But file assets/test.png does not exist, so removing splash and re-adding it again will probably fail - please check it.

I am not also sure if packages/flutter_splash_tizen/example/tizen/shared/res/test.png file will work as a splash screen, as the tizen-manifest file uses: assets/test.png.